README.md

```diff
@@ -175,6 +202,13 @@ In general, every development board has the following basic features:
 
 ## 3D Models
 
+- 3D preview of the *[SRA Board 2022(New link yet to be attached)](https://a360.co/3c1Rjyv)*
+
+    1. The complete 3D model (.stl) file of [SRA Board 2022]()
```

Comment:
Link Point 1

Reply:
Instead of Point 2 mentioning other components model
Mention LSA and MPU models.